src/Apps/Artwork/ArtworkApp.tsx

```diff
@@ -24,9 +26,54 @@ import { Media } from "Utils/Responsive"
 
 export interface Props {
   artwork: ArtworkApp_artwork
+  tracking?: TrackingProp
 }
 
+declare const window: any
+@track()
 export class ArtworkApp extends React.Component<Props> {
+  // TODO: Move the below tracking, which consists of:
```

Comment:
@damassi as you pointed out with the Order app, we can do some basic pageview tracking in Reaction (but can't yet pass these properties over).

Since we also need a custom event (which I think makes sense here in `componentDidMount` for the special 'Viewed Product' track), I decided to do the pageview like that here as well (rather than use that wrapper).